automation/jenkins_build.sh

```diff
@@ -60,6 +60,11 @@ deploy_build () {
 
 	test "$SLUG" = "edge" && return
 
+	if [ -z "$_deploy_artifact" ] && [ -z "$_deploy_flasher_artifact" ]; then
+		echo "[WARN] No deploy artifacts defined."
+		return
+	fi
+
 	cp -v "$YOCTO_BUILD_DEPLOY/kernel_modules_headers.tar.gz" "$_deploy_dir"
```

Comment:
Do these devices don't create kernel modules headers archive?

Reply:
These device types are meant to run only in a container so they rely on the host's kernel.